### PR TITLE
change parseCommaSeparatedValue to parseCommaOrSemicolonSeparatedValue

### PR DESCRIPTION
**Pull request by Uzlopak**
Resolves https://github.com/fastify/workflows/issues/58

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by simoneb — COMMENTED** [review]
Can you update the readme accordingly?

**Comment by Uzlopak** [comment]
@simoneb 
done :)

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ steps:
   - uses: fastify/github-action-merge-dependabot@v3.0.0
     with:
       github-token: ${{ secrets.GITHUB_TOKEN }}
-      exclude: 'react,fastify'
+      exclude: 'react;fastify'
```
Comment (simoneb) [inline README.md:98]: I'd keep this like the original which I believe is more common

```suggestion
      exclude: 'react,fastify'
```

Comment (Uzlopak) [reply README.md]: no problem

**Review by simoneb — APPROVED** [review]

**Pull request merged**